larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -80,34 +80,10 @@ void* Init(TfLiteContext* context, const char* buffer, std::size_t length) {
   conv_params->dilation_height_factor = m["dilation_height_factor"].AsInt32();
   conv_params->dilation_width_factor = m["dilation_width_factor"].AsInt32();
 
-  // reading padding
-  if (m["padding"].ToString() == "VALID" ||
-      m["padding"].ToString() == "valid") {
-    conv_params->padding_type = kTfLitePaddingValid;
-  } else if (m["padding"].ToString() == "SAME" ||
-             m["padding"].ToString() == "same") {
-    conv_params->padding_type = kTfLitePaddingSame;
-  } else {
-    context->ReportError(context, "Invalid padding attribute.");
-    return conv_params;
-  }
+  conv_params->padding_type = (TfLitePadding)m["padding"].AsInt32();
+  conv_params->fused_activation_function =
+      (TfLiteFusedActivation)m["fused_activation_function"].AsInt32();
```

Comment:
Not sure if we need to do some error checking here.

Reply:
I'd opt for adding an `LCE_ENSURE_PARAM(conv_params, context, !m["fused_activation_function"].IsNull());` to the list thats slightly above these lines.

Reply:
Good point, added a check in f092d5b31a33c0d64d886869269526a7f7e5061c